### PR TITLE
Added table showing support for algo 15

### PR DESCRIPTION
**Pull request by matsduf**

### CODE REVIEW

**Comment by matsduf** [comment]
@sandoche2k, table is completed, please review again.

File: README.md [hunk 1]
```diff
@@ -130,11 +130,7 @@ Debian 10        | yes
 FreeBSD 11.3     | yes
 FreeBSD 12.1     | yes
 Ubuntu 16.04     | ???
```
Comment (mattias-p) [inline README.md]: zonemaster/zonemaster-engine#678 claims that 16.04 does not support algo 15. Should this row too be updated to reflect that?

Comment (matsduf) [reply README.md]: @mattias-p, you reviewed without my last patch yesterday. Please review again.

**Comment by matsduf** [comment]
@sandoche2k, please review again.

File: README.md (hunk 1 above)
Comment (mattias-p) [reply README.md]: Weird. I wonder how that happened.

**Pull request merged**